src/main/java/AnimeGirl.java

```diff
@@ -0,0 +1,22 @@
+public class AnimeGirl implements Domesticatable, Tradable{
+    public AnimeGirl() {
```

Comment:
Perhaps needs a private variable to store all the information that this class has